apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -1275,6 +1955,93 @@ module.exports = {
       name: "settings",
       label: "Settings",
       files: [
+        {
+          name: "general",
+          label: "General",
+          file: `${APP_DIRECTORY}content/settings/general.md`,
+          fields: [
+            {
+              label: "SEO",
+              name: "seo",
+              widget: "object",
+              fields: [
+                {
+                  label: "Title",
+                  name: "title",
+                  widget: "string",
+                  required: false,
+                },
+                {
+                  label: "Description",
+                  name: "description",
+                  widget: "string",
+                  required: false,
+                },
+                {
+                  label: "Open Graph",
+                  name: "openGraph",
+                  widget: "object",
+                  fields: [
+                    {
+                      label: "Title",
+                      name: "title",
+                      widget: "string",
+                      required: false,
+                    },
+                    {
+                      label: "Description",
+                      name: "description",
+                      widget: "string",
+                      required: false,
+                    },
+                    {
+                      label: "Images",
+                      name: "images",
+                      widget: "list",
+                      required: false,
+                      fields: [
+                        {
+                          name: "url",
+                          label: "URL",
+                          widget: "string",
+                        },
+                        {
+                          name: "width",
+                          label: "Width",
+                          widget: "string",
+                        },
+                        {
+                          name: "height",
+                          label: "Height",
+                          widget: "string",
+                        },
+                        {
+                          name: "alt",
+                          label: "Alt Text",
+                          widget: "string",
+                        },
+                      ],
+                    },
+                  ],
+                },
+                {
+                  label: "Twitter",
+                  name: "twitter",
+                  widget: "object",
+                  fields: [
+                    {
+                      label: "Handle",
+                      name: "handle",
```

Comment:
```suggestion
                      label: "@Site",
                      name: "site",
```

You should be setting site handle in settings page.